fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/ExpressionConverter.java

```diff
@@ -134,16 +130,16 @@ public Expr visitLiteral(Literal literal, PlanTranslatorContext context) {
 
     // TODO: Supports for `distinct`
     @Override
-    public Expr visitFunctionCall(FunctionCall function, PlanTranslatorContext context) {
+    public Expr visitBoundFunction(BoundFunction function, PlanTranslatorContext context) {
         List<Expr> paramList = new ArrayList<>();
-        for (Expression expr : function.getFnParams().getExpressionList()) {
+        for (Expression expr : function.getArguments()) {
             paramList.add(visit(expr, context));
```

Comment:
DefaultExpressionVisitor.visit always returns null
```suggestion
            paramList.add(expr.accept(this, context));
```